core/src/test/scala/cromwell/core/labels/LabelSpec.scala

```diff
@@ -13,21 +13,30 @@ class LabelSpec extends FlatSpec with Matchers {
   val goodLabelKeys = List(
     "cromwell-root-workflow-id",
     "cromwell-11f2468c-39d6-4be3-85c8-32735c01e66b",
-    "just-the-right-length-just-the-right-length-just-the-right-leng"
+    "just-the-right-length-just-the-right-length-just-the-right-leng",
+    "11f2468c-39d6-4be3-85c8-32735c01e66b",
+    "0-cromwell-root-workflow-id",
+    "cromwell-root-workflow-id-",
+    "0-cromwell-root-workflow-id-",
+    "Cromwell-root-workflow-id",
+    "now valid 255 character key-at vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa",
+    "!@#$%^&*()_+={}[]:;'<>?,./`~"
   )
 
   val goodLabelValues = List(
     "11f2468c-39d6-4be3-85c8-32735c01e66b",
-    ""
+    "",
+    "!@#$%^&*()_+={}[]:;'<>?,./`~",
+    "now valid 255 character value-at vero eosd accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa",
```

Comment:
Should be "eos" (no 'd' at the end) - though if this is only a long string test and not a match test then that isn't really an issue, except for my Ancient Roman OCD.